src/transformStateWithClones.js

```diff
@@ -6,8 +6,60 @@
  *
  * @return {Object[]}
  */
+
 function transformStateWithClones(state, actions) {
-  // write code here
+  const transformedState = [];
+
```

Comment:
```suggestion
 let transformedState;
```
You can create additional variable and modify it